SwiftWisdomTests/VersionTests.swift

```diff
@@ -35,4 +35,16 @@ class VersionTests: XCTestCase {
         
         XCTAssert(highest > middle)
     }
+    
+    func testVersionLessThan() {
+        let left = Version(string: "1.0.0")!
+        let right = Version(string: "0.5.2")!
+        XCTAssertFalse(left < right)
+    }
```

Comment:
Maybe something with no patch? One with a revision? One with gibberish in the initializer string? `33,64/f4`